nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -326,6 +333,8 @@ let
 
     ${sslConf}
 
+    ${if cfg.package.luaSupport then luaSetPaths else ""}
```

Comment:
@nagy this ends up being slightly annoying for me... do you mind rewriting this as
```
${optionalString (cfg.package.luaSupport or false) luaSetPaths}
```

Reply:
I have adapted this in another pull request that I opened recently on this topic again.

https://github.com/NixOS/nixpkgs/pull/108511

Reply:
Thank you very much @nagy!